### PR TITLE
BNF-1367 Rebrand

### PR DESCRIPTION
**Pull request by barkertron**
https://nicedigital.atlassian.net/browse/BNF-1367

### CODE REVIEW

File: gatsby/src/components/DrugSections/IndicationsAndDose/IndicationsAndDoseContent/IndicationsAndDoseContent.module.scss [hunk 1]
```diff
@@ -59,33 +62,33 @@
...
   /* stylelint-disable function-parentheses-space-inside */
   // Use patterned colours to distinguish patient groups
   &.neonate::before {
     background: repeating-linear-gradient(
       -45deg,
-      core.$nds-colour-nice-teal-2,
-      core.$nds-colour-nice-teal-2 1px,
-      core.$nds-colour-nice-ice-teal 1px,
-      core.$nds-colour-nice-ice-teal 12px
+      colours.$indication-dose-neonate-1,
```
Comment (ediblecode) [inline gatsby/src/components/DrugSections/IndicationsAndDose/IndicationsAndDoseContent/IndicationsAndDoseContent.module.scss:73]: Lovely

File: gatsby/src/components/SiteDistinction/SiteDistinction.module.scss [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
   z-index: 101; // Has to be visible over the sticky section nav (z: 100) and back to top (z:99)
 
   &.bnf {
-    border-top-color: $bnf-brand-colour;
+    border-top-color: settings.$bnf-brand-colour;
   }
 
   &.bnfc {
-    border-top-color: $bnfc-brand-colour;
+    border-top-color: settings.$bnfc-brand-colour;
```
Comment (ediblecode) [inline gatsby/src/components/SiteDistinction/SiteDistinction.module.scss:20]: We might as well get rid of the settings file and move into the colour tokens file?

File: gatsby/gatsby-ssr.tsx [hunk 3]
```diff
@@ -21,6 +21,21 @@ export const onPreRenderHTML = ({
...
+	// Add Google Fonts
+	components.push(
+		<>
+			<link
+				rel="preconnect"
+				href="https://fonts.gstatic.com"
+				crossOrigin="crossOrigin"
+			/>
+			<link
+				href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&family=Lora:ital,wght@0,400;0,600;1,400;1,600&display=swap"
```
Comment (ediblecode) [inline gatsby/gatsby-ssr.tsx:33]: Can get just use Lora 600 and not 400 please?

File: gatsby/src/styles/_colours.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
+@use '@nice-digital/nds-core/scss/colours/tokens/global';
```
Comment (ediblecode) [inline gatsby/src/styles/_colours.scss:1]: This file is glorious

File: gatsby/gatsby-ssr.tsx [hunk 5]
```diff
@@ -21,6 +21,21 @@ export const onPreRenderHTML = ({
 		></script>
 	);
 
+	// Add Google Fonts
+	components.push(
```
Comment (ediblecode) [inline gatsby/gatsby-ssr.tsx:25]: If we're changing our font loading mechanism from self-hosted with link preloads, then please could you remove the link headers from web.config as they're no longer needed?

File: gatsby/src/styles/global.scss [hunk 6]
```diff
@@ -1,32 +1,25 @@
...
 
 $nds-include-spacing-modifiers: false;
 $nds-output-text-classes: false;
 $nds-output-visibility-classes: false;
 
 @import './settings';
 
-@import './lato';
-
 @import '~@nice-digital/design-system/scss/base';
```
Comment (ediblecode) [inline gatsby/src/styles/global.scss:10]: I don't think we need the ~ any more?

File: gatsby/src/styles/global.scss [hunk 7]
```diff
@@ -1,32 +1,25 @@
-@use '@nice-digital/nds-core/scss/core';
+@use '@nice-digital/nds-core/scss/spacing';
+@use '@nice-digital/nds-core/scss/utils';
 
 $nds-include-spacing-modifiers: false;
```
Comment (ediblecode) [inline gatsby/src/styles/global.scss:4]: I think these variables have changed names, and can be configured with the import below so they're like:

```
@use '@nice-digital/design-system/scss/base' with (
  $include-spacing-modifiers: false,
  $output-text-classes: false,
  $output-visibility-classes: false
);
```

File: gatsby/src/styles/global.scss [hunk 8]
```diff
@@ -1,32 +1,25 @@
...
 $nds-output-text-classes: false;
 $nds-output-visibility-classes: false;
 
 @import './settings';
```
Comment (ediblecode) [inline gatsby/src/styles/global.scss:8]: Can we change these to `@use` please?

**Review by ediblecode — CHANGES_REQUESTED** [review]

**Review by ediblecode — DISMISSED** [review]

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**